src/accelerate/utils/launch.py

```diff
@@ -101,12 +101,12 @@ def prepare_simple_launcher_cmd_env(args: argparse.Namespace) -> Tuple[List[str]
 
     current_env["OMP_NUM_THREADS"] = str(args.num_cpu_threads_per_process)
 
-    if args.cpu or args.use_cpu:
-        if args.ipex_enabled:
-            current_env["IPEX_ENABLED"] = str(args.ipex_enabled).lower()
+    if is_ipex_available():
```

Comment:
So would my understanding be wrong here, that we can use `xpu` as a training paradigm and it's seperate from ipex?

Reply:
So the plugin is the same , only changes w.r.t devices. In case of no details, if cpu + ipex is there it goes into the condition , even if it should go the xpu .  Since the underlying code plugin is the same , depending on devices it should segregate.